grammatical typo (#6761)
wso2/docs-apim: en/docs/use-cases/streaming-tutorials/working-with-kafka.md

## Introduction

> The Streaming Integrator can consume from a Kafka topic as well as publish to a Kafka topic in a streaming manner.

Reviewer: ```suggestion
The Streaming Integrator can consume from a Kafka topic and publish to a Kafka topic in a streaming manner.
```
